script.js

```diff
@@ -79,42 +79,61 @@ function shuffleCards() {
 
 function compareCards(e) {
   let clickedCard = e.target;
-  clickedCard.classList.add("flipped");
+  selectCard(clickedCard);
 
-  let flippedCards = document.querySelectorAll(".flipped");
+  let selectedCards = document.querySelectorAll(".selected");
```

Comment:
Renamed to make it clear

Reply:
Renamed variable - makes cleared for all